src/user/user.entity.ts

```diff
@@ -166,6 +166,26 @@ export class DbUser extends BaseEntity {
   })
   readonly twitter_username?: string;
 
+  @ApiModelProperty({
+    description: "LinkedIn URL",
+    example: "https://www.linkedin.com/company/open-sauced/",
```

Comment:
the example repo would need to be `/in` for profiles and not companies
```suggestion
    example: "https://www.linkedin.com/in/open-sauced/",
```

Reply:
I guess you are catching this, but we'd want to encourage `/in` over everything else